packages/core/src/components/DataTable/helpers/expandDataList.ts

```diff
@@ -0,0 +1,38 @@
+import { SortDirectionType } from 'react-virtualized';
+import sortList from './sortList';
+import { IndexedParentRow, IndexedChildRow, ExpandedRow } from '../types';
+
+/*  Iterate over the sortedDataList in state to flatten out children stashed in metadata */
+export default function expandDataList(
+  sortedDataList: IndexedParentRow[],
+  expandedRows: Set<number>,
+  sortBy: string,
```

Comment:
Is there a type for this?

Reply:
For `sortBy`? No, there's one for `sortDirection`, but `sortBy` is just any column name so it's just any string.